integration_tests/e2e/profile/overview.cy.ts

```diff
@@ -50,7 +51,7 @@ context('Profile Overview Page', () => {
       .hasLddAssessmentsRecorded()
       .hasConditionsRecorded('Acquired brain injury')
       .hasStrengthsRecorded('Numeracy skills')
-      .hasNoSupportRecommendationsRecorded()
+      .hasNoSupportStrategiesRecorded()
```

Comment:
👍 for the renaming here; It's important that our naming/terminology etc is consistent throughout the codebase, so well done for remembering/spotting this 👍 👏